argo-cd-apps/base/quality-dashboard.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: quality-dashboard
+
+spec:
+  project: default
+
+  source:
+    path: components/quality-dashboard
+    repoURL: https://github.com/redhat-appstudio/quality-dashboard.git
```

Comment:
Shouldn't this be `https://github.com/redhat-appstudio/infra-deployments.git` ?

Reply:
fixed